app/services/authn_context_resolver.rb

```diff
@@ -14,7 +24,20 @@ def resolve
     if vtr.present?
       selected_vtr_parser_result_from_vtr_list
     else
-      acr_result_with_sp_defaults
+      acr_results
+    end
+  end
+
+  # Expose the resolved IAL ACR value
+  def asserted_ial_value
+    if acr_results.biometric_comparison?
+      Saml::Idp::Constants::IAL2_BIO_REQUIRED_AUTHN_CONTEXT_CLASSREF
+    elsif acr_results.ialmax?
```

Comment:
based on some tests that are breaking, i think i made this too simple -- it needs to be updated to see if a user has been verified or not, and return the IAL1 or IAL2 context classref